src/commands/threat-feed/cmd-threat-feed.ts

```diff
@@ -51,6 +55,32 @@ const config: CliCommandConfig = {
     Options
       ${getFlagListOutput(config.flags, 6)}
 
+    This feature requires an Enterprise Plan with Threat Feed add-on. Please
+    contact sales@socket.dev if you would like access to this feature.
+
+    Valid filters:
+
+      - c       Do not filter
+      - u       Unreviewed
+      - fp      False Positives
+      - tp      False Positives and Unreviewed
+      - mal     Malware and Possible Malware [default]
+      - vuln    Vulnerability
+      - anom    Anomaly
+      - secret  Secret
+      - joke    Joke / Fake
+      - spy     Telemetry
+      - typo    Typo-squat
```

Comment:
☝️ Can we rank these alphabetically or by severity (list the severity)

Reply:
I mean, in a way they are.
I'm okay to alpha them but not sure that's any better than what it was before. I'm not even sure about severity, I don't think that makes much sense for these filter words.